### ISSUE TITLE
Cracks on top bolts on [a]tap_center_r6

### ISSUE THREAD

**Issue by SinisterRj** [VoronDesign/Voron-Tap#50]
Trying to debug some ressonance issues on my printer I was checking the overal boolts torques and accidentaly break this piece. But looking at it, I felt that this part is verry week at this point, with only 3 walls from here the bolts make contact do the end of the part. I was able to increase this number to 5 without reducing the clearences to the other parts. And in my modification I was abble to cincundate the entire rail carrige, also increasing the stiffness of the part a little bit. The new part is printed and it is working on my printer.

![tap issue 2](https://user-images.githubusercontent.com/81830673/218560795-dd91d819-5259-4ec0-882b-78681010f5e8.jpg)
![tap issue 1](https://user-images.githubusercontent.com/81830673/218560801-5eea4c50-5cc5-4b32-a9bc-9582e9ccb1c7.jpg)
![tap](https://user-images.githubusercontent.com/81830673/218560802-19d5c579-af4a-4cd4-b6e2-d594eee28193.png)
![tap2](https://user-images.githubusercontent.com/81830673/218561298-b845ed62-f4fd-4ed2-9b68-4b3dfe769f74.png)
![tap3](https://user-images.githubusercontent.com/81830673/218561301-631191c4-5635-4fff-bad5-a9b564e34e2a.png)

**Comment by raymondh2** [comment]
Hi. Could you please provide what material and color the part was printed in?

**Comment by SinisterRj** [comment]
It was printed in Dental White ABS MG94 from GTMax3D (a brasilian 3d printer manufacturer and one of the best filaments in Brazil)

**Comment by nemgrea** [comment]
unfortunately white filaments are known to be less robust compared to their equivalent darker colored counterparts due to the colorants needed for getting white filament.

**Comment by SinisterRj** [comment]
Yes, I know. The new part I printed on black. But, anyway, it gave me an opportunity to improve a weak point of the part.
